docs/reference/validation_operators/perform_action_list_validation_operator.rst

```diff
@@ -0,0 +1,80 @@
+
+
+PerformActionListValidationOperator validates each batch in its `run` method's `assets_to_validate` argument against the expectation suite included within that batch.
+
+Then it invokes a list of configured actions on every validation result.
+
+Each action in the list must be an instance of NamespacedValidationAction
+class (or its descendants). Read more about actions here: :ref:`actions`.
+
+The init command includes this operator in the default configuration file.
+
+
+Configuration
+--------------
+
+An instance of PerformActionListValidationOperator is included in the default configuration file `great_expectations.yml` that `great_expectations init` command creates.
+
+.. code-block:: yaml
+
+    perform_action_list_operator: # this is the name you will use when you invoke the operator
+        class_name: PerformActionListValidationOperator
+
+        # the operator will call the following actions on each validation result
+        # you can remove or add actions to this list. See the details in the actions
+        # reference
+        action_list:
+          - name: store_validation_result
+            action:
+              class_name: StoreAction
+              target_store_name: local_validation_result_store
+          - name: send_slack_notification_on_validation_result
+            action:
+              class_name: SlackNotificationAction
+              # put the actual webhook URL in the uncommitted/config_variables.yml file
+              slack_webhook: ${validation_notification_slack_webhook}
+             notify_on: all # possible values: "all", "failure", "success"
+              renderer:
+                module_name: great_expectations.render.renderer.slack_renderer
+                class_name: SlackRenderer
+
+
+Invokation
+-----------
+
+This is an example of invoking an instance of a Validation Operator from Python:
+
+.. code-block:: python
+
+    results = context.run_validation_operator(
+        assets_to_validate=[batch0, batch1, ...],
+        run_identifier="some_string_that_uniquely_identifies_this_run",
+        validation_operator_name="perform_action_list_operator",
+    )
+
+* `assets_to_validate` - a list that specifies the data assets that the operator will validate. The members of the list can be either batches (which means that have data asset identifier, batch kwargs and expectation suite identifier) or a triple that will allow the operator to fetch the batch: (data asset identifier, batch kwargs, expectation suite identifier) using this method: :py:meth:`great_expectations.data_context.ConfigOnlyDataContext.get_batch`
+* run_identifier - pipeline run id, a timestamp or any other string that is meaningful to you and will help you refer to the result of this operation later
+* validation_operator_name you can instances of a class that implements a Validation Operator
+
+The `run` method returns an object that looks like this:
+
+.. code-block:: json
+
+    {
```

Comment:
This needs to be real JSON to avoid sphinx complaining. Can we just quote the strings or something?